index.js

```diff
@@ -37,7 +38,11 @@ module.exports = {
     global.EmberENV = EmberENV;
 
     var Compiler = require(templateCompilerPath);
-    var PrecompileInlineHTMLBarsPlugin = HTMLBarsInlinePrecompilePlugin(Compiler.precompile); // jshint ignore:line
+    var templateCompilerFullPath = require.resolve(templateCompilerPath);
+    var templateCompilerCacheKey = fs.readFileSync(templateCompilerFullPath, { encoding: 'utf-8' });
```

Comment:
Ya, but upon restart it will blow the async cache properly (which was not true previously).  So its a tad better, but could use more work.